server/requirements.txt

```diff
@@ -2,5 +2,6 @@ gunicorn==19.6.0
 honcho==0.6.6
 pyodbc==3.0.10
 unidecode==0.04.19
+celery==3.1.25
```

Comment:
Thanks @petrjasek , I have changed that. I have also included the other packages as fixed versions.